extensions/text.js

```diff
@@ -666,6 +700,8 @@
     toCase(args, util) {
       const string = args.STRING.toString();
       const textCase = args.TEXTCASE.toString();
+      let sum = "";
+      let capflag = false;
```

Comment:
abbreviating names does not help. you can write out a fully descriptive name

Reply:
Renamed these to runningText and sentenceCapitalFlag